contrib/ansible/roles/master/tasks/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Install kubernetes
+  action: "{{ ansible_pkg_mgr }}"
+  args:
+        name: kubernetes-master
```

Comment:
Right now `kubernetes-master` and `kubernetes-node` packages are only in the testing repo for CentOS7.

We should just call that out in the README, if the assumption is the repositories for whatever distro you are using are already setup.

```
[root@kube-minion-test-01 ~]# yum --showduplicates list kubernetes* | expand
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: cosmos.cites.illinois.edu
 * extras: centos.mia.host-engine.com
 * updates: centos.mbni.med.umich.edu
Installed Packages
kubernetes.x86_64               0.18.0-0.2.git5520386.el7         @virt7-testing
kubernetes-master.x86_64        0.18.0-0.2.git5520386.el7         @virt7-testing
kubernetes-node.x86_64          0.18.0-0.2.git5520386.el7         @virt7-testing
Available Packages
kubernetes.x86_64               0.9.0-0.3.git96af0c3.el7          extras        
kubernetes.x86_64               0.15.0-0.3.git0ea87e4.el7         extras 
```